ironmon_tracker/data/RandomizerLog.lua

```diff
@@ -2,11 +2,12 @@
 RandomizerLog = {}
 
 RandomizerLog.Patterns = {
-	RandomizerVersion = "Randomizer Version:%s*([%d%.]+)%s*$", -- Note: log file line 1 does NOT start with "Rando..."
+	-- Acount for possible -dev or -beta in version string
+	RandomizerVersion = "Randomizer Version:%s*([%d%.]+).*$", -- Note: log file line 1 does NOT start with "Rando..."
```

Comment:
This change is simply to have the pattern match the whole line if there are any characters after the version number, could be including in the capturing group. Likely not a needed change

Reply:
This is a simply enough change that can catch any future Randomizer updates without needing to be so strict. As such, I think it's a good change and should be included.